a10_neutron_lbaas/a10_context.py

```diff
@@ -108,6 +111,19 @@ def __exit__(self, exc_type, exc_value, traceback):
         super(A10WriteContext, self).__exit__(exc_type, exc_value, traceback)
 
 
+class A10ReplayContext(A10WriteContext):
+
+    def __init__(self, *args, **kwargs):
+        self._appliance = kwargs.pop('appliance', None)
```

Comment:
I really don't like the use of 'appliance' and 'device' together in the same place (I hate both, actually.) But we can change its name post-commit.